subscription/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Use python image
+FROM python:3.13-slim
+
+#Set environment variables
+ENV AWS_REGION us-east-1
+ENV QUEUE_URL https://sqs.us-east-1.amazonaws.com/832706493240/cmr-internal-subscriptions-queue-sit
```

Comment:
Seems like these should be environment dependent, not hardcoded

Reply:
I have change this to use the Docker ARG value to be able to pass in environment variables during the build process.